cmd/oras/root/pull.go

```diff
@@ -130,40 +130,44 @@ func runPull(ctx context.Context, opts pullOptions) error {
 	dst.AllowPathTraversalOnWrite = opts.PathTraversal
 	dst.DisableOverwrite = opts.KeepOldFiles
 
-	desc, pulledEmpty, err := doPull(ctx, src, dst, copyOptions, &opts)
+	desc, skippedLayers, err := doPull(ctx, src, dst, copyOptions, &opts)
 	if err != nil {
 		if errors.Is(err, file.ErrPathTraversalDisallowed) {
 			err = fmt.Errorf("%s: %w", "use flag --allow-path-traversal to allow insecurely pulling files outside of working directory", err)
 		}
 		return err
 	}
-	if pulledEmpty {
-		fmt.Println("Downloaded empty artifact")
+
+	// suggest oras copy for pulling layers without annotation
+	if skippedLayers > 0 {
+		fmt.Printf("Skipped pulling layers without file name in %q\n", ocispec.AnnotationTitle)
+		fmt.Printf("Use 'oras copy %s --to-oci-layout <layout-dir>' to pull all layers.\n", opts.RawReference)
+	} else {
+		fmt.Println("Pulled", opts.AnnotatedReference())
+		fmt.Println("Digest:", desc.Digest)
 	}
-	fmt.Println("Pulled", opts.AnnotatedReference())
-	fmt.Println("Digest:", desc.Digest)
 	return nil
 }
 
-func doPull(ctx context.Context, src oras.ReadOnlyTarget, dst oras.GraphTarget, opts oras.CopyOptions, po *pullOptions) (ocispec.Descriptor, bool, error) {
+func doPull(ctx context.Context, src oras.ReadOnlyTarget, dst oras.GraphTarget, opts oras.CopyOptions, po *pullOptions) (ocispec.Descriptor, int, error) {
 	var configPath, configMediaType string
 	var err error
 	if po.ManifestConfigRef != "" {
 		configPath, configMediaType, err = fileref.Parse(po.ManifestConfigRef, "")
 		if err != nil {
-			return ocispec.Descriptor{}, false, err
+			return ocispec.Descriptor{}, 0, err
 		}
 	}
 
 	var tracked track.GraphTarget
-	dst, tracked, err = getTrackedTarget(dst, po.TTY, "Downloading", "Downloaded ")
+	dst, tracked, err = getTrackedTarget(dst, po.TTY, "Downloading", "Pulled")
```

Comment:
Better make output aligned to `Skipped    `
```suggestion
	dst, tracked, err = getTrackedTarget(dst, po.TTY, "Downloading", "Pulled     ")
```